eslint.config.js

```diff
@@ -156,10 +168,26 @@ module.exports = [
         }
     },
 
+    // JSON files
+    {
+        name: "eslint/json",
+        files: ["**/*.json"],
```

Comment:
Maybe we can also lint .c8rc:

```suggestion
        files: ["**/*.json", ".c8rc"],
```